scripts/prettier_check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+export TERM=xterm-color
+
+SEARCH_PATH=${@:-'src/**/*.md'}
+
+npx prettier --write --loglevel warn $SEARCH_PATH
+
+git --no-pager diff --color --exit-code src/ README.md
```

Comment:
This is a clever solution, but IMO having this script always modify files in place would be too much of a surprise/gotcha for anyone who might run this script locally.

Suggestions:
* Have this script accept general prettier args, and if no args are passed in default to `--check src/**/*.md`.
* Rename the script `prettier.sh` (having "check" in the name would imply always doing `--check`).
* Change the CI to call this script with `--write --loglevel warn ${{ needs.get-changed-files.outputs.changed_files }}`, followed by running the `git diff` command.

Reply:
Agreed - would definitely not be good to just have this overwrite. I might move this code over to `prettier_fix.sh`, which already overwrites